code/src/containers/forms/product-form/components/additional-info/AdditionalInfo.tsx

```diff
@@ -0,0 +1,110 @@
+import { Controller, ControllerRenderProps } from "react-hook-form";
+
+import { productCategories } from "@/containers/forms/product-form/ProductForm.constants";
+import {
+  ProductFormAdditionalInfoSectionProps,
+  ProductFormValues
+} from "@/containers/forms/product-form/ProductForm.types";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppCheckbox from "@/components/app-checkbox/AppCheckbox";
+import AppFormHelperText from "@/components/app-form-helper-text/AppFormHelperText";
+import AppInput from "@/components/app-input/AppInput";
+import AppMenuItem from "@/components/app-menu-item/AppMenuItem";
+import AppSelect from "@/components/app-select/AppSelect";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+const AdditionalInfo = ({
+  register,
+  errors,
+  control
+}: ProductFormAdditionalInfoSectionProps) => {
+  const categoryErrorElement = errors.category && (
+    <AppFormHelperText className="product-form__category-select-helper" error>
+      {errors.category.message}
+    </AppFormHelperText>
+  );
+
+  const selectControllerRenderFunction = ({
+    field: handlers
+  }: {
+    field: ControllerRenderProps<ProductFormValues>;
+  }) => (
+    <>
+      <AppSelect
+        fullWidth
+        label="productForm.inputLabel.category"
+        inputProps={{
+          className: "product-form__category-select"
+        }}
+        error={Boolean(errors.category)}
+        data-testid="product-form-category-select"
+        {...handlers}
+      >
+        {productCategories.map((item) => (
+          <AppMenuItem value={item.id} key={item.id}>
+            <AppTypography
+              className="product-form__category-select-label"
+              translationKey={item.label}
+            />
+          </AppMenuItem>
+        ))}
+      </AppSelect>
+      {categoryErrorElement}
+    </>
+  );
+
+  return (
+    <AppBox className="product-form__container product-form__additional-info-section">
+      <AppBox className="product-form__header">
+        <AppTypography
+          component="h1"
+          variant="h3"
+          translationKey="productForm.section.additionalInformation.title"
+          className="product-form__header-title"
+        />
+      </AppBox>
+      <AppBox className="product-form__body">
+        <AppBox className="product-form__quantity-price-container">
+          <AppInput
+            fullWidth
+            labelTranslationKey="productForm.inputLabel.price"
+            inputProps={{ min: 0, step: "any" }}
```

Comment:
Step `'any'` is a default value so it can be omitted, isn't it?

Reply:
Nope, for number input step default value is 1. The problem with default value is that it doesn't allow float numbers like 999.99 or 500.50, but it should be allowed for price. Any makes float numbers allowed and at the same time uses default step 1.